build.gradle.kts

```diff
@@ -806,6 +806,18 @@ allprojects {
                 // Do not publish "root" project. Java plugin is applied here for DSL purposes only
                 return@configure
             }
+
+            repositories {
+                maven {
+                    name = "ApacheSnapshot" //  optional target repository name
+                    url = uri("https://repository.apache.org/content/repositories/snapshots")
+                    credentials {
+                        username = (project.properties["nexusSnapshotUser"] ?: "").toString()
+                        password = (project.properties["nexusSnapshotPassword"] ?: "").toString()
+                    }
+                }
+            }
```

Comment:
The repository is already configured, so there's no need to add one more

Reply:
https://calcite.apache.org/docs/howto.html#making-a-snapshot

Reply:
Can you please answer my questions: 

Where is this defined in the build files? What are the username and password property names?

Reply:
@jacques-n , I understand it might be frustrating, however, could you please try finding the answer at howto.html?

Calcite releases are pushed to ASF Nexus for a long time, so the repository must be declared.

Of course, I can answer your questions, however, in that case, I would like to remove howto.html since it seems to add no value.

Reply:
Closing conversation due to disrespectful behavior